README.md

```diff
@@ -563,6 +563,14 @@ docker run -v $PWD/pyrdp_output:/home/pyrdp/pyrdp_output -p 3389:3389 gosecure/p
 Make sure that your destination directory is owned by a user with a UID of 1000, otherwise you will get permission denied errors.
 If you are the only non-root user on the system, usually your user will be assigned UID 1000.
 
+#### Logging the host IP address
+
+If you want PyRDP to log the host IP address in its logs, you can set the HOST_IP environment variable when using `docker run`:
+
+```
+docker run -p 3389:3389 -e HOST_IP=192.168.1.9 gosecure/pyrdp pyrdp-mitm.py 192.168.1.10
```

Comment:
I'm not sure about the HOST_IP name. Does this align to something you've seen used elsewhere with containers?

Reply:
No, we could name it anything else honestly.

Reply:
After thinking about it a little more, `HOST_IP` is fine.